pkg/apis/istio/common/v1alpha1/string.go

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
```

Comment:
Did this get generated or you did this by hand? this is for my learning :sweat_smile:

Reply:
All was done by hand :(